lmp/tokenizer/_base_dict_tokenizer.py

```diff
@@ -277,9 +302,21 @@ def decode(
                 tokens except unknown word's token.
                 See class docstring for more details on special tokens.
 
+        Raises:
+            TypeError:
+                When `token_ids` is not instance of `List[int]`.
+                When `remove_special_tokens` is not instance of `bool`.
```

Comment:
When `token_ids` is not instance of `List[int]` or `remove_special_tokens` is not instance of `bool`.